src/index.ts

```diff
@@ -409,6 +410,20 @@ function setupOpenMRS() {
           }
         )
       },
+      {
+        id: "edit-patient-details-button",
+        slot: "patient-actions-slot",
+        load: getAsyncLifecycle(
+          () =>
+            import(
+              "./ui-components/custom-overflow-menu/custom-items/edit-patient-details"
```

Comment:
This whole thing seems to be best placed in patient registration.

After all the URL (and the respective parts) etc. are known only in patient registration.

Please put the component there.